content/news/playtest-20150524.md

```diff
@@ -0,0 +1,55 @@
+It has been a month of busy development. Time to get back to testing.
+
+The mayor changes this time around include:
+
+  - A whole new in-game map editor with all the benefits that come from it.
+  - Improved system integration with `openra://` server join URLs and instant `.orarep` playback.
+  - The Dune 2000 mod got building upgrades which re-creates the tech tree of the original.
+  - Several enhanced classic missions have been added to the Tiberian Dawn mod.
+  - In the Red Alert mod you will now be notified when enemy units appear in your sight.
```

Comment:
No, also RA. https://github.com/OpenRA/OpenRA/pull/7849 and yes it is a bit weak. Not much new for players to discover in this changeset, but I tried to balance the per mod coverage as demanded.